# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/wifi/moz.build
# Commit: 1cf92ef74fb4
# Full Hash: 1cf92ef74fb45d3f5b62eb3f7b0fd7c35a2d4f4f
# Author: David Parks <daparks@mozilla.com>
# Date: 2023-05-11 04:06:39
# Regressor Bug: 1810421
# File Overlap Count: 2
# Description:
#   Bug 1810421: Only scan wifi when the network changes or when on mobile r=necko-reviewers,kershaw
#   
#   Simplifies the concurrent operations of wifi scanning and reduces the
#   frequency in common cases.  Wifi scanning when on mobile is reduced from
#   every 5 seconds to every minute.  Wifi scans will also happen whenever
# ==============================================================================

diff -r afd62e82fe83 -r 1cf92ef74fb4 netwerk/wifi/moz.build
--- a/netwerk/wifi/moz.build	Wed May 10 19:46:29 2023 +0000
+++ b/netwerk/wifi/moz.build	Wed May 10 19:46:29 2023 +0000
@@ -56,6 +56,8 @@
     ]
     CXXFLAGS += CONFIG["MOZ_DBUS_GLIB_CFLAGS"]
 
+TEST_DIRS += ["gtest"]
+
 FINAL_LIBRARY = "xul"
 
 include("/ipc/chromium/chromium-config.mozbuild")